tests/core/ckeditor/basepathglobal.js

```diff
@@ -10,7 +10,7 @@
 		folderPath = path + testDir.slice( 0, testDir.lastIndexOf( '/' ) ),
 		expectedEditorPath = '/apps/ckeditor/',
 		query = CKEDITOR.timestamp ? '?t=' + CKEDITOR.timestamp : '',
-		secondDomainName = 'sub.ckeditor.dev',
+		secondDomainName = 'sub.' + window.location.hostname,
```

Comment:
This change results in `sub.tests.ckeditor.test` in standard setup and therefore – 404 error in console. What's strange, tests are still passing despite this fact.

Reply:
Because the tests only checks if paths are properly created (on `iframe load event`) and not checking if something was loaded properly (and `/apps/ckeditor.js` which contains `getUrl` method is always loaded from main domain):

![image](https://user-images.githubusercontent.com/1061942/32329304-f362c3a6-bfdc-11e7-9d42-d3b6262de8b0.png)

versus (as `loader.js` failed to load)

![image](https://user-images.githubusercontent.com/1061942/32329336-0d64b1a6-bfdd-11e7-91fa-2a52a4e371ce.png)

It passes on Chrome no matter the subdomain used. And fails on Safari... no matter the subdomain used.

I think I need take another look on #1061.

Reply:
Small correction on Safari it fails only with subdomain ending with `.dev` is used.

Reply:
More confusing than I thought on Safari...

Fails for:
* [sub.]ckeditor.dev
* [sub.]ckeditor.test
* [sub.]ckeditor.local
* [sub.]ckeditor.com

Passes for:
* [sub.]ckeditor.custom
* [sub.]ckeditor.pl
* [sub.]ckeditor.bcn

So from what I assume, Safari treats `.dev`, `.test`, `.local` alwasy as non local, but globally registered domains? (because for such domains, like `ckeditor.com`, this test fails also in Chrome).

So tests passes on Chrome/Safari for non-existing domains or locally defined domain which contains proper CKEditor files (unless local domain in Safari ends with `.dev`, `.test`, `.local`).

@Comandeer does it make any sense? (quite confusing for me IMHO)

I pushed fix with the custom domain `sub.ckeditor.custom`.

Reply:
No, it doesn't make any sense 🤔

Reply:
Or maybe it does… https://security.stackexchange.com/a/14805